documentation/src/main/asciidoc/topics/ref_jgroups_system_properties.adoc

```diff
@@ -108,7 +103,7 @@ The following system properties apply to `default-jgroups-kubernetes.xml`:
 
 .Google Cloud Platform
 
-The following system properties apply to `default-jgroups-google.xml`:
+The following system properties apply only to `default-jgroups-google.xml`:
 
 [%header,cols="1,3,2,1"]
 |===
```

Comment:
can you check column "required" in `jgroups.google.bucket_name`? It is mentioning `dns_query` but I'm not sure if it is related.

Reply:
fixed